apps/webapp/web/router.ex

```diff
@@ -36,7 +36,8 @@ defmodule Webapp.Router do
     get "/login", PageController, :login
     get "/docs", PageController, :docs
     get "/docs/index", DocController, :index
-    get "/docs/:filename", DocController, :show
+    get "/docs/:filename", PageController, :docs
```

Comment:
Wow, I understand, I will make the change. It looks much better and interesting.